apps/lfx-one/src/app/modules/dashboards/executive-director/components/revenue-impact-drawer/revenue-impact-drawer.component.html

```
@@ -0,0 +1,171 @@
+            </div>
+          }
+        </div>
+      } @else {
+        <div class="flex items-center justify-center h-[80px] text-sm text-gray-400" data-testid="revenue-impact-drawer-engagement-empty">
+          Deal pipeline data not yet available
+        </div>
+      }
+    </div>
+
+    <!-- Attribution Model Comparison -->
+    @if (data().attributionModels.linear + data().attributionModels.firstTouch + data().attributionModels.lastTouch > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)